rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
 channel = "stable"
-targets = ["wasm32-unknown-unknown"]
+targets = ["wasm32-unknown-unknown", "wasm32v1-none"]
```

Comment:
we could build parser to target `wasm32v1-none` too

Reply:
`wasm32v1-none` is `#![no_std]` target. if this is not possible, use current target.